lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -740,6 +745,19 @@ sub schema_fixup
         $data->{'artist_credit'} = MusicBrainz::Server::Entity::ArtistCredit->new( { names => \@credits } );
     }
 
+    if (defined $data->{aliases}) {
+        my @aliases = map {
+            MusicBrainz::Server::Entity::Alias->new(
+                sort_name => $_->{sort_name} || '',
+                locale => $_->{locale} || '',
+                name => $_->{name} || '',
```

Comment:
`//` can be used to only fall back to `''` if the LHS is `undef` (rather than any falsey value). I guess it would only matter if the name or sort name of the alias is `'0'` (unfortunately, `('0' || '') eq ''` evaluates to `1`).
```suggestion
                sort_name => $_->{sort_name} // '',
                locale => $_->{locale} // '',
                name => $_->{name} // '',
```

Reply:
Thanks, done.